### PR TITLE
matching changes for edge-proto reorg

### PR DESCRIPTION
**Pull request by gainsley**
A bunch of changes to match the edge-proto file reorg which split the proto files into separate definitions for each API group, resulting in separate Java classes here.

Also here:
- fix for port number of custom DME URI being ignored in SDK Demo App
- fix for "missing android:exported" build failures, fixed by upgrading androidx.test.ext:junit:1.1.2 to androidx.test.ext:junit:1.1.3
- fixes for some lint failures around UI objects missing horizontal or vertical constraints

### CODE REVIEW

File: android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/qoe/QoeMapActivity.java [hunk 1]
```diff
@@ -96,6 +96,8 @@
```
Comment (brucedarmstrong) [inline android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/qoe/QoeMapActivity.java:96]: Just wanted to point out that this Activity was removed from the app's main menu when DT stopped supporting their QOS API. Thought about removing the code completely, but there was a bunch of Google Maps API code that I wanted to continue to have access to.

**Review by brucedarmstrong — APPROVED** [review]
Looks good. If you can't find an Android guy, become an Android guy. :)

One other thing I realized is that the overall project name is still MobiledgeXSDKDemo. I can't remember if we discussed changing this and there was some reason not to.

**Comment by brucedarmstrong** [comment]
Oh, heck. I'm logged in to the wrong account. I'll review it from the right one.

**Review by brucedarmstrong — APPROVED** [review]
👍

**Review by brucearmstrong — APPROVED** [review]
👍

**Comment by gainsley** [comment]
Thanks Bruce! Good point about renaming it. Also, the source code packages are all named com.mobiledgex.xyz...Would we need to rename those before we can push these new apps to the Android store?

**Comment by brucearmstrong** [comment]
Yes, the package would have to be renamed too.

**Pull request merged**